### ISSUE TITLE
request to add nessus

### ISSUE THREAD

**Issue by pranj4** [BlackArch/blackarch#4178]
can you please include nessus in blackarch beacuse while installing from aur i has lots of issues in it

**Comment by noraj** [comment]
Hi,

- The reason why nessus is not in AL or BA directly is because of Tenable License Agreement that forbid the distribution, so it can't be served prepackaged, you have to build it for yourself so it can't go farther than having an install script in the AUR
- I personally am the maintainer of the nessus PKGBUILD in the AUR and there is absolutely no issue with it. The only reason for the 404 often is that Tenable has an unique download link for each version of nessus and remove link for older versions so at each new release the lnik must be updated.

TL;DR: that won't happen
